examples/pylab_examples/agg_buffer.py

```diff
@@ -13,7 +13,7 @@
 try:
     from PIL import Image
 except ImportError:
-    raise SystemExit("PIL must be installed to run this example")
+    raise SystemExit("pillow must be installed to run this example")
```

Comment:
Sure -- I can capitalize.  It's not clear: they have inconsistent capitalization on their website: https://python-pillow.github.io/